source/examples/generated/TransactionsTest.snippet.transaction.kt

```diff
@@ -0,0 +1,39 @@
+// Set up the MongoDB client and get the collection
+suspend fun performTransaction(client: MongoClient) {
+    client.startSession().use { session ->
+        session.startTransaction() // Start the transaction
```

Comment:
Typically I've always put comments on their own lines above the line they're meant to describe/annotate
```suggestion
        // Start the transaction
        session.startTransaction()
```